README.rst

```diff
@@ -380,14 +380,47 @@ logins on the same device, you do not wish the old user to receive messages whil
 Via DRF, any creation of device with an already existing registration ID will be transformed into an update.
 If done manually, you are responsible for deleting the old device entry.
 
+Using custom FCMDevice model
+----------------------------
+
+If there's a need to store additional information or change type of fields in the FCMDevice model.
+You could simple override this model. It could be fully your own model or you could inheriting from AbstractFCMDevice 
+
+In your ``your_app/models.py``:
+
+.. code-block:: python
+
+    import uuid
+    from django.db import models
+    from fcm_django.models import AbstractFCMDevice
+
+
+    class CustomDevice(AbstractFCMDevice, models.Model):
+        # fields could be overwritten
+        id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+        # could be added new fields
+        updated_at = models.DateTimeField(auto_now=True)
+
+In your ``settings.py``:
+
+.. code-block:: python
+
+    FCM_DJANGO_FCMDEVICE_MODEL = "your_app.CustomDevice"
+
+Note: This functionality based on `Swapper <https://pypi.org/project/swapper/>`_ that based on functionality 
+that allow to use a `custom User model <https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#substituting-a-custom-user-model>`_.
+So this functionality have the same limitations. 
+The most is important limitation it is that is difficult to start out with a default (non-swapped) model 
+and then later to switch to a swapped implementation without doing some migration hacking.
```

Comment:
A line on whether there's any specialty with migrations if a custom model is used and how to kickstart it would be nice

Reply:
You could copy [branch](https://github.com/Akay7/fcm-django-web-demo/tree/swappedModels) to your repo and I could mention that diff between two branches in this repo.

Reply:
I would rather not maintain two branches, maybe we can make the fcm-django-web-demo have two separate examples

Reply:
For me personally it's way harder to maintain two branches. Someone opens an issue it needs to be fixed in two places in two separate branches, these two branches eventually cannot be reconciliated anymore, epecially if there's PRs involved

Reply:
Okay I will make copy of project in the main branch.